src/java/org/apache/cassandra/index/sai/IndexContext.java

```diff
@@ -607,7 +607,7 @@ public boolean supports(Operator op)
         if (op.isLike() || op == Operator.LIKE) return false;
         // Analyzed columns store the indexed result, so we are unable to compute raw equality.
         // The only supported operator is ANALYZER_MATCHES.
-        if (isAnalyzed || op == Operator.ANALYZER_MATCHES) return isAnalyzed && op == Operator.ANALYZER_MATCHES;
+        if (op == Operator.ANALYZER_MATCHES) return isAnalyzed && op == Operator.ANALYZER_MATCHES;
```

Comment:
Can't this be just `if (op == Operator.ANALYZER_MATCHES) return isAnalyzed`?

Reply:
sure!